Makefile

```diff
@@ -78,6 +79,7 @@ install:
 	if [ ! -f $(VIRTUAL_ENV)/bin/python3 ]; then python3 -m venv $(VIRTUAL_ENV); fi
 	$(VIRTUAL_ENV)/bin/python3 -m pip install --upgrade -r requirements/dev.txt
 	$(VIRTUAL_ENV)/bin/python3 manage.py migrate
+	if [ ! -d $(MEDIA_ROOT) ]; then mkdir -p $(MEDIA_ROOT); fi
```

Comment:
if you clone repo and `make install`and `make watch`it won't work as the footer contrib tag has no where to put the downloaded footer, by creating media folder on install it solves this problem and also should solve it for the CI to

Reply:
No need for the `if` check, the `-p` option only creates the dir if it doesn't exist.

Reply:
@m4ra fixed :)